app/controllers/answer_like_controller.rb

```diff
@@ -0,0 +1,20 @@
+class AnswerLikeController < ApplicationController
+  before_filter :require_user
+
+  def show
+    render :json => Answer.find(params[:id]).cached_likes
+  end
+
+  def likes
+    @answer = Answer.find(params[:aid])
+    if @answer.liked_by(current_user.uid)
+      AnswerSelection.set_likes(current_user.uid,@answer.id,false)
```

Comment:
This is really small, but would you mind separating commas with spaces, like `function(one, two, three)`? Thanks!